src/main/resources/templates/admin/plantingSiteMap.html

```diff
@@ -173,6 +193,27 @@ <h1 th:if="${site.plantingZones.isEmpty()}">This planting site has no map data.<
                     'line-color': 'white',
                 }
             });
+
+            const plotClickHandler = (e) => {
+                const props = e.features[0].properties;
+                const description = [
+                        `<b>${props.name} (${props.id})</b>`,
```

Comment:
Another option is `<h4>` instead of `<b>`, comes with extra top/bottom margin and will be in bold. Can remove 201 with this, and it may leave more room for the 'x'.